src/app/types/transcription-urgency.interface.ts

```diff
@@ -1,4 +1,5 @@
 export interface TranscriptionUrgency {
   transcription_urgency_id: number;
   description: string;
+  priority_order?: number;
```

Comment:
Not sure this is optional?

Reply:
@deanbullock-hmcts suggested that it be optional

Reply:
Argh, it isn't marked as required in the openapi spec, so I guess this is fine.